Bottom: e11eab01d2afaa0bdf6e58a3ad81b7f65fe23e78
Top:    3c98e6ff297326e1cfc0e489cd651ed0c96259fb
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-18 22:04:28 +0000

Refresh of action-c-adjust-some-whitespac

---

diff --git a/src/action.c b/src/action.c
index 8a1e357..6c3ab45 100644
--- a/src/action.c
+++ b/src/action.c
@@ -2679,8 +2679,7 @@ ActionDisplay (int argc, char **argv, Coord childX, Coord childY)
 
 	case F_CycleClip:
 	  notify_crosshair_change (false);
-	  if TEST_FLAG
-	    (ALLDIRECTIONFLAG, PCB)
+	  if TEST_FLAG (ALLDIRECTIONFLAG, PCB)
 	    {
 	      TOGGLE_FLAG (ALLDIRECTIONFLAG, PCB);
 	      PCB->Clipping = 0;
